app/uk/gov/hmrc/transitmovements/models/formats/PresentationFormats.scala

```diff
@@ -30,6 +31,7 @@ trait PresentationFormats extends CommonFormats {
   implicit val messageFormat: Format[Message]                                   = Json.format[Message]
   implicit val movementFormat: Format[Movement]                                 = Json.format[Movement]
   implicit val departureWithoutMessagesFormat: Format[DepartureWithoutMessages] = Json.format[DepartureWithoutMessages]
```

Comment:
I don't think we still need departureWithoutMessagesFormat/DepartureWithoutMessages? If so, please delete

Reply:
yes, it should be deleted once this API - _**GET   /traders/:EORI/movements/departures/:movementId**_  is implemented

Reply:
Why not now? We're no longer using it right as it has been replaced with MovementWithoutMessages?

Reply:
thought of doing this earlier, as this is still being used for the other API _get single departure by departureid_  GET /traders/:EORI/movements/departures/:movementId

It would be good if we delete this as part of the below ticket 
https://jira.tools.tax.service.gov.uk/browse/CTCP-1489

Reply:
@prashanthmamidi Changing it to MovementWithoutMessages for that endpoint shouldn't affect anything since they are both the same model but with different names. I will write a note in the ticket so that the person implementing it for arrivals remembers to update the respective departures endpoint too.